src/base/bittorrent/session.cpp

```diff
@@ -1998,6 +1998,11 @@ void Session::increaseTorrentsPriority(const QStringList &hashes)
         torrentQueuePositionUp(torrent->nativeHandle());
         torrentQueue.pop();
     }
+
+    // Save fastresume for all torrents in the queue
+    for (TorrentHandle *const torrent : torrents())
```

Comment:
Maybe this piece of code should become a function?

Reply:
One tiny problem, the shortest name i can think of that is adequately descriptive is a full sentence... i mean when it will only save resume data for torrents that shifted in the queue.

Reply:
@thalieht:  `QueueShiftResumeSaver`? 

`QueueShiftResumeHandler`?

Reply:
I mean the function name, what it does i.e. saveFastResumesForReprioritizedTorrents. Hey now that i wrote it down it doesn't seem _too_ long, i'll go with that if no one has a better one.

Reply:
Just throwing out ideas, or how about rename handleTorrentPriorityChanged() -> handleTorrent**s**PriorityChanged() and move this loop into it, then call the function where needed.